model/metrics/container.yaml

```diff
@@ -0,0 +1,57 @@
+groups:
+  - id: metric.container.cpu.usage
+    type: metric
+    metric_name: container.cpu.usage
```

Comment:
This likely needs a set of attributes, or do you expect that these metrics would report against a single container as the resource?

Reply:
Yeap, you are right those will need to be accompanied with attributes like `container.name` etc.